package.json

```diff
@@ -8,7 +8,7 @@
     "DSL"
   ],
   "scripts": {
-    "test": "mocha test/*"
+    "test": "mocha test/reader/*"
```

Comment:
It's a mistake as there will be more tests in the folder than just the reader subfolder.